### PR TITLE
C-3330: Move concat-md to devDependencies

### PR DESCRIPTION
**Pull request by tk26**
## Description

[Pull Request Description Here]

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> `Fix [#1]()`

### CODE REVIEW

File: packages/react-toast/package.json [hunk 1]
```diff
@@ -21,9 +21,11 @@
     "types": "tsc --declaration --outDir typings/ --emitDeclarationOnly --declarationMap --allowJs false --checkJs false"
   },
   "license": "MIT",
+  "devDependencies": {
+    "concat-md": "^0.3.5"
+  },
   "dependencies": {
-    "@trycourier/react-provider": "^0.1.12-beta.0",
-    "concat-md": "^0.3.5",
+    "@trycourier/react-provider": "^0.1.12-beta.1",
```
Comment (suhasdeshpande) [inline packages/react-toast/package.json:28]: did we do `yarn install`? I would expect some changes to lock files no?

Comment (tk26) [reply packages/react-toast/package.json:28]: Yep, no changes to locks

File: lerna.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "packages": ["packages/*"],
-  "version": "0.1.12-beta.0",
+  "version": "0.1.12-beta.1",
```
Comment (rileylnapier) [inline lerna.json:3]: i don't think we need to version this

Comment (tk26) [reply lerna.json]: good call - undone.

**Review by rileylnapier — APPROVED** [review]

**Review by scarney81 — APPROVED** [review]

**Pull request merged**